api-reference/50 Common/Object Structures/positionConfig/collision/collision.md

```diff
@@ -5,48 +5,127 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies how to move the widget if it overflows the screen.
+Specifies how to resolve collisions between the overlay and the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) elements.
```

Comment:
I'm not sure what you mean by "resolve collisions".
Do you mean:
Specifies how to prevent the overlap of [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) elements.

If not, please clarify.

Reply:
How about something like this:
Specifies how to resolve a collision - when an overlay element exceeds its [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary).